### ISSUE TITLE
Add UI feedback to UIStepper

### ISSUE THREAD

**Issue by chrisballinger** [Burning-Man-Earth/iBurn-iOS#35]
needs background/highlighted image

**Comment by OnlyInAmerica** [comment]
Should probably also increase magnitude of each click. It takes ~4s hold or 9 clicks to change the filter from 5m walk to 10m walk.

**Comment by chrisballinger** [comment]
Yeah I agree, right now it's +/- 50 meters, but 100 meters would probably be better.
